src/nl/help/faq/README.md

```diff
@@ -24,16 +24,31 @@ Als je problemen hebt met de app, kijk dan op deze pagina voordat je een bericht
 
 ---
 
-### App loopt vast bij het opstarten
-Als je de **Publieke build** gebruikt, controleer dan of je iOS 13.4+ of iPadOS 13.4+ gebruikt. Als je de **Beta / Patron build** gebruikt, stuur een bericht op het Discord #supporter-kanaal en ping Paper (In het Engels).
+### De app loopt vast bij het opstarten
+Zorg ervoor dat je iOS 13.4+ of iPadOS 13.4+ gebruikt.
+
+Als je de **Beta / Patron build** gebruikt kan het zijn dat deze onstabiel is. Als je de juiste iOS of iPadOS versie voor je apparaat gebruikt en nog steeds crasht, kan het zijn dat je een al bekende bug tegen bent gekomen. Neem contact met ons op via het Discord #supporter-chatkanaal en iemand zal je proberen te helpen met het probleem.
```

Comment:
Could a space or a hyphen be missing? You did write `#support-kanaal` in getting-started.
```suggestion
Als je de **Beta / Patron build** gebruikt kan het zijn dat deze onstabiel is. Als je de juiste iOS of iPadOS versie voor je apparaat gebruikt en nog steeds crasht, kan het zijn dat je een al bekende bug tegen bent gekomen. Neem contact met ons op via het Discord #supporter-chat kanaal en iemand zal je proberen te helpen met het probleem.
```